stock_picking_mass_assign/wizard/check_assign_all.py

```diff
@@ -1,135 +1,76 @@
+from openerp.exceptions import UserError
 
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
-from openerp.osv import orm, fields
-from openerp.tools.translate import _
-
-
-class check_assign_all(orm.TransientModel):
+class CheckAssignAll(models.TransientModel):
     _name = 'stock.picking.check.assign.all'
     _description = 'Delivery Orders Check Availability'
 
-    _columns = {
-        'check_availability': fields.boolean(
-            'Check Availability', help="""check this box if you want to"""
-            """ check the availability of the selected Delivery Orders."""),
-        'force_availability': fields.boolean(
-            'Force Availability', help="""check this box if you want to"""
-            """ force the availability of the selected Delivery Orders."""),
-        'process_picking': fields.boolean(
-            'Deliver', help="""check this box if you want to"""
-            """ deliver all the selected Delivery Orders.\n You'll not have"""
-            """ the possibility to realize a partial delivery.\n If you want"""
-            """ to do that, please do it manually on the Delivery Order"""
-            """ form."""),
-        'create_invoice': fields.boolean(
-            'Create Invoices/Refunds', help="""check this box if you want to"""
-            """ create Invoices or Refunds for all the selected Delivery"""
-            """ Orders."""),
-    }
-
-    def _default_force_availability(self, cr, uid, context=None):
-        context = context or {}
-        return context.get('force_availability', False)
-
-    def _default_process_picking(self, cr, uid, context=None):
-        context = context or {}
-        return context.get('process_picking', False)
-
-    _defaults = {
-        'check_availability': True,
-        'force_availability': _default_force_availability,
-        'process_picking': _default_process_picking,
-    }
-
-    def check(self, cr, uid, ids, context=None):
-        context = context or {}
-        picking_obj = self.pool['stock.picking']
-        process_wizard_obj = self.pool['stock.partial.picking']
-        picking_ids = context.get('active_ids')
-        wizard_id = ids[0]
+    @api.multi
+    def _default_force_availability(self):
```

Comment:
[Please put these methods below the field declarations](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field)